pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -38,7 +38,7 @@ type remoteImageService struct {
 
 // NewRemoteImageService creates a new internalapi.ImageManagerService.
 func NewRemoteImageService(endpoint string, connectionTimeout time.Duration) (internalapi.ImageManagerService, error) {
-	klog.V(3).Infof("Connecting to image service %s", endpoint)
+	klog.V(3).InfoS("Connecting to image service", "imageManagerService", endpoint)
```

Comment:
I'd guess we'd rather use a standard label here like "endpoint" rather than special casing this - @serathius ?